From ce16c459ac0d4a0f51ee4fe2b4ca543e8b81f4c5 Mon Sep 17 00:00:00 2001
From: Ting Liu <ting.liu@freescale.com>
Date: Tue, 18 Aug 2015 18:48:58 +0800
Subject: [PATCH] glibc-2.20 support

Signed-off-by: Ting Liu <ting.liu@freescale.com>
---
 configure.in | 21 ++++++++++++++-------
 1 file changed, 14 insertions(+), 7 deletions(-)

diff --git a/configure.in b/configure.in
index c64aa70..7e0d72a 100644
--- a/configure.in
+++ b/configure.in
@@ -934,13 +934,20 @@ case "${GLIBC_VERSION}" in
 	DEFAULT_SUPP="$srcdir/glibc-2.34567-NPTL-helgrind.supp ${DEFAULT_SUPP}"
 	DEFAULT_SUPP="$srcdir/glibc-2.X-drd.supp ${DEFAULT_SUPP}"
 	;;
-     2.21)
-    AC_MSG_RESULT(2.21 family)
-    AC_DEFINE([GLIBC_2_19], 1, [Define to 1 if you're using glibc 2.21.x])
-    DEFAULT_SUPP="glibc-2.X.supp ${DEFAULT_SUPP}"
-    DEFAULT_SUPP="$srcdir/glibc-2.34567-NPTL-helgrind.supp ${DEFAULT_SUPP}"
-    DEFAULT_SUPP="$srcdir/glibc-2.X-drd.supp ${DEFAULT_SUPP}"
-    ;;
+      2.20)
+        AC_MSG_RESULT(2.20 family)
+        AC_DEFINE([GLIBC_2_20], 1, [Define to 1 if you're using glibc 2.20.x])
+        DEFAULT_SUPP="glibc-2.X.supp ${DEFAULT_SUPP}"
+        DEFAULT_SUPP="$srcdir/glibc-2.34567-NPTL-helgrind.supp ${DEFAULT_SUPP}"
+        DEFAULT_SUPP="$srcdir/glibc-2.X-drd.supp ${DEFAULT_SUPP}"
+        ;;
+      2.21)
+        AC_MSG_RESULT(2.21 family)
+        AC_DEFINE([GLIBC_2_21], 1, [Define to 1 if you're using glibc 2.21.x])
+        DEFAULT_SUPP="glibc-2.X.supp ${DEFAULT_SUPP}"
+        DEFAULT_SUPP="$srcdir/glibc-2.34567-NPTL-helgrind.supp ${DEFAULT_SUPP}"
+        DEFAULT_SUPP="$srcdir/glibc-2.X-drd.supp ${DEFAULT_SUPP}"
+        ;;
      darwin)
 	AC_MSG_RESULT(Darwin)
 	AC_DEFINE([DARWIN_LIBC], 1, [Define to 1 if you're using Darwin])
-- 
1.9.2

